src/main.js

```diff
@@ -521,7 +569,15 @@ function createShellAppLauncherMetaCallProxy(interfaceMetadata,
                                                 wrapped,
                                                 args);
     };
-    return createMetaCallProxy(interfaceMetadata, asyncCallFunc, syncCallFunc);
+    let proxy = createMetaCallProxy(interfaceMetadata, asyncCallFunc, syncCallFunc);
+
+    // Also add method to directly launch the app if required
+    proxy.LaunchApplicationDirectlyRemote = function(timestamp, callback) {
+      return shellProxy.LaunchRemote(appId, timestamp, callback);
+    };
+    proxy.LaunchApplicationDirectlySync = function(timestamp) {
```

Comment:
This is unused

Reply:
It is unused but included here for completeness sake (I'm mirroring the API that `wrapDBusProxy` gives us in gjs)

Reply:
I don't think that it's worth mirroring that API, or that we should hide the `shellProxy` indirection here.

Reply:
Hmm okay. I'd rather not have to create the `shellProxy` object again from the app launcher ... perhaps I could return both as an object and then use object destructuring to get the one I want.